converters/fromarray/layout.ts

```diff
@@ -0,0 +1,153 @@
+let layoutConverters = [];
+layoutConverters[Action.Insert] = insertFromArray;
+layoutConverters[Action.Remove] = removeFromArray;
+layoutConverters[Action.Move] = moveFromArray;
+layoutConverters[Action.AttributeUpdate] = attributeUpdateFromArray;
+layoutConverters[Action.CharacterDataUpdate] = cdataUpdateFromArray;
+layoutConverters[Action.Scroll] = scrollFromArray;
+layoutConverters[Action.Input] = inputFromArray;
```

Comment:
Could we do something more generic that would just send the changed fields and a bit mas that encodes what those fields are? Seems simpler and would have less code to maintain.